includes/notes/class-wc-admin-notes-welcome-message.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * WooCommerce Admin: Welcome to WooCommerce Admin Note
+ *
+ * Adds welcome notes to store alerts area.
+ *
+ * @package WooCommerce Admin
+ */
+
+defined( 'ABSPATH' ) || exit;
+/**
+ * WC_Admin_Notes_Welcome_Message.
+ */
+class WC_Admin_Notes_Welcome_Message {
+	const NOTE_NAME = 'woocommerce-admin-welcome-note';
+
+	/**
+	 * Creates a note for welcome message.
+	 */
+	public static function add_welcome_note() {
+
+		// First, see if we've already created this kind of note so we don't do it again.
+		$data_store = WC_Data_Store::load( 'admin-note' );
+		$note_ids   = $data_store->get_notes_with_name( self::NOTE_NAME );
+		if ( ! empty( $note_ids ) ) {
```

Comment:
If I'm not wrong, we have remove this check, otherwise we are never adding this note because we have previously created `WC_Admin_Notes_Historical_Data`.

Reply:
Will not it check that Note Name(which is wc-admin-welcome-note) is available? if not then and only it will add the note. But maybe it will not need because it will work on install so. removed it.

Reply:
I missed this message, my bad!  :man_facepalming:  You were absolutely right!

I just created a PR that adds this code back (#2063). Thank you for pointing this out and my apologizes for missing this message.